examples/notebooks/models/thermal-models.ipynb

```diff
@@ -196,16 +198,16 @@
     "\n",
     "along with boundary conditions\n",
     "\n",
-    "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = h_{cn}(T-T_\\infty),$$\n",
+    "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = \\frac{L_{cn}h_{cn} + (L_n+L_s+L_p+L_{cp})h_{edge}}{L_{cn}+L_n+L_s+L_p+L_{cp}}(T-T_\\infty),$$\n",
     "at the negative tab,\n",
-    "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = h_{cp}(T-T_\\infty),$$  \n",
+    "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = \\frac{(L_{cn}+L_n+L_s+L_p)h_{edge}+L_{cp}h_{cp}}{L_{cn}+L_n+L_s+L_p+L_{cp}}(T-T_\\infty),$$  \n",
     "at the positive tab, and\n",
     "$$ -\\lambda_{eff} \\nabla_\\perp T \\cdot \\boldsymbol{n} = h_{edge}(T-T_\\infty),$$  \n",
     "elsewhere.\n",
     "\n",
     "Here the heat source term is averaged in the x direction so that $\\bar{Q}=\\bar{Q}(y,z)$. The parameter $\\lambda_{eff}$ is the effective thermal conductivity, computed as \n",
     "\n",
-    "$$ \\rho_{eff} = \\frac{\\sum_k \\rho_k c_{p,k} L_k}{\\sum_k L_k}.$$\n",
+    "$$ \\lambda_{eff} = \\frac{\\sum_k \\lambda_k L_k}{\\sum_k L_k}.$$\n",
```

Comment:
should we change the name of the parameter in pybamm accordingly?

Reply:
the name is correct in pybamm - I had just written completely the wrong equation in the notebook!

Reply:
looks like I copied it from above to save me writing out the sum again and forgot to change rho to lambda